src/pages/blog/the-perfect-react-bootcamp-student.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Are YOU the Perfect React Bootcamp Student?
+date: 2019-01-25
+subtitle: The React Bootcamp is catered to devs looking to upskill, but is it right for you?!
+author: richard
+imageUrl: https://firebasestorage.googleapis.com/v0/b/reactjsacademy-react.appspot.com/o/blog%20post%20images%2FperfectReactStudent%2FIMG_0414.jpg?alt=media&
+order: 15
+---
+
+We’ve been teaching React since 2016 and in that time we’ve often been asked what makes the perfect ReactJS Academy student. We’ve been asked what base knowledge you need, especially. So today we’re going to answer a few FAQs to make sure you get the most out of React Bootcamp - and we’ll have some snippets from past students along the way!
+
+![image alt text](https://firebasestorage.googleapis.com/v0/b/reactjsacademy-react.appspot.com/o/blog%20post%20images%2FperfectReactStudent%2Fimage_1.jpg?alt=media&)
+
+## What do ReactJS Academy students learn?
```

Comment:
@ric9176 @shakersinthedark I found out that adding an anchor in the titles helps for link building. Example, you can have at the bottom a "wrap up / recap" that includes a link to those titles. Then when you run the auto import of the post in Medium we get a lot of link to our website. Example, the conclusion at the bottom (https://reactjs.academy/blog/react-forms-controlled-and-uncontrolled-components/) has a link to the section "Form library" https://reactjs.academy/blog/react-forms-controlled-and-uncontrolled-components/#form-library. And it's also the same in Medium https://medium.com/leanjs/react-forms-controlled-uncontrolled-components-7842a7284dfa, though the Medium links take the user to the section in our blog